### PR TITLE
Update symptom checker flow

### PR DESCRIPTION
**Pull request by devinjameson**
Why: as a user, I would like to be able to tell the app my symptoms and be given a link
where I can find a place nearby to get tested for COVID-19.

This commit:
- Adds content to the `AtRiskRecommendation` screen
- Always navigates users to the `AtRiskRecommendation` screen regardless of their symptoms
- Updates the content of the `CheckIn` screen to also include a button to find a test center

gif

### CODE REVIEW

File: src/SymptomChecker/AtRiskRecommendation.tsx [hunk 1]
```diff
@@ -19,6 +26,10 @@ const AtRiskRecommendationScreen: FunctionComponent = () => {
     navigation.navigate(SymptomCheckerStackScreens.SymptomChecker)
   }
 
+  const handleOnPressFindTestCenter = () => {
+    Linking.openURL("https://google.com")
```
Comment (devinjameson) [inline src/SymptomChecker/AtRiskRecommendation.tsx]: Placeholder.

File: src/SymptomChecker/CheckIn.tsx [hunk 2]
```diff
@@ -149,10 +150,23 @@ const FeelingNotWellContent: FunctionComponent<FeelingNotWellContentProps> = ({
 }) => {
   const { t } = useTranslation()
 
+  const handleOnPressFindTestCenter = () => {
+    Linking.openURL("https://google.com")
```
Comment (devinjameson) [inline src/SymptomChecker/CheckIn.tsx]: Placeholder.

File: src/ConfigurationContext.tsx [hunk 3]
```diff
@@ -73,6 +76,7 @@ const ConfigurationProvider: FunctionComponent = ({ children }) => {
         displayReportAnIssue,
         displaySelfAssessment,
         displaySymptomChecker,
+        findATestCenterUrl,
```
Comment (aledustet) [inline src/ConfigurationContext.tsx:79]: This means this URL is not being localized. This is perfectly fine, maybe worth asking if it is supported by the authority or if that's a desire.

**Review by aledustet — APPROVED** [review]
Looking goood

File: src/ConfigurationContext.tsx (hunk 3 above)
Comment (johnschoeman) [reply src/ConfigurationContext.tsx:79]: for now we dont need to support localization of the link here.  We can add it as an enhancement if turns out to be a requirement

File: src/SymptomChecker/AtRiskRecommendation.tsx [hunk 4]
```diff
@@ -71,8 +112,32 @@ const style = StyleSheet.create({
   cancelButtonInnerContainer: {
     padding: Spacing.medium,
   },
+  image: {
+    resizeMode: "contain",
+    width: "97%",
+    height: 300,
```
Comment (johnschoeman) [inline src/SymptomChecker/AtRiskRecommendation.tsx]: Does this fixed height force a scroll on small format screens?

File: src/SymptomChecker/symptoms.ts [hunk 5]
```diff
@@ -6,10 +6,7 @@ export enum HealthAssessment {
...
 
 export const determineHealthAssessment = (
-  symptoms: Symptom[],
+  _symptoms: Symptom[],
 ): HealthAssessment => {
-  if (symptoms.length > 3) {
-    return HealthAssessment.AtRisk
-  }
-  return HealthAssessment.NotAtRisk
+  return HealthAssessment.AtRisk
```
Comment (johnschoeman) [inline src/SymptomChecker/symptoms.ts]: We probably want `if (symptoms.length > 0)` here.

**Review by johnschoeman — APPROVED** [review]
LGTM

File: src/SymptomChecker/symptoms.ts (hunk 5 above)
Comment (devinjameson) [reply src/SymptomChecker/symptoms.ts]: Just curious why?

File: src/SymptomChecker/AtRiskRecommendation.tsx (hunk 4 above)
Comment (devinjameson) [reply src/SymptomChecker/AtRiskRecommendation.tsx]: Yeah it does, I'll make this smaller and reduce the text size to avoid scrolling shenanigans.

**Pull request merged**